Sources/Network/ProductService.swift

```diff
@@ -128,7 +128,8 @@ class ProductService: ProductApi {
             //url.append(contentsOf: "?fields=" + summaryFields.joined(separator: OFFJson.FieldsSeparator) + OFFJson.FieldsSeparator + languageFields.joined(separator: OFFJson.FieldsSeparator))
             url.append(contentsOf: "?fields=" + OFFJson.summaryFields.joined(separator: OFFJson.FieldsSeparator))
         } else {
-            url.append(contentsOf: "?fields=" + OFFJson.allFields.joined(separator: OFFJson.FieldsSeparator) + OFFJson.languageCodes)
+            let allFieldsWithProductAttributes = OFFJson.allFields + [ProductAttributesJson.AttributeGroupsKey]
+            url.append(contentsOf: "?fields=" + allFieldsWithProductAttributes.joined(separator: OFFJson.FieldsSeparator) + OFFJson.languageCodes)
         }
```

Comment:
One more thing I have to do here before I finish:
- Add language prefix to the `attribute_groups` field when building the api query here. E.g. when lang=English, `url.append("?fields=" + ... + "attribute_groups_en")`

Reply:
Remind me: you need to add the language code, so that you get the strings in the app language?

Reply:
Now, I am using a new method `ProductAttributesJson.attributeGroupsLang()` to append the language code from `Bundle.main.currentLocalization` to the api key `attribute_groups` before making the request. 
There is potential for an error here: if `attributeGroupsLang()` returns "" then the attribute groups parameter will be "attribute_groups_" with a trailing "_"

Reply:
Also, I don't like how I'm manually appending the "attribute_groups" parameter to the `OFFJson.allFields` array. 
If we want to make api requests that exclude 'Attribute Groups' in the future on certain conditions, I need to know how that should be handled. Otherwise, they are included in every request (in an "unfiltered" call; i.e. `isSummary = false`)

But that could be a different PR.